backend/src/models/listingModel.js

```diff
@@ -1,14 +1,53 @@
 const mongoose = require('mongoose');
 
 const ListingSchema = new mongoose.Schema({
-    listerID: { type: mongoose.Schema.Types.ObjectId, required: true },
-    name: { type: String, required: true, minlength: 5, maxlength: 30 },
-    description: { type: String },
-    rental_price: { type: Number, required: true },
-    date: { type: Date, required: true },
-    pet_friendly: { type: Boolean },
-    active: { type: Boolean, default: true, required: true },
-    location: { type: String, coordinates: [Number], required: true },
+    listerID: { 
```

Comment:
should we just rename to userId for simplicity?

Reply:
Yes!